joschahenningsen/TUM-Live
# Feature/split view fullscreen

!! This should be merged after https://github.com/joschahenningsen/TUM-Live/pull/891

### Motivation and Context
Currently its not possible to fullscreen both videos of the split view player.

### Description
The standard behavior of the fullscreen button is overwritten to now make both videos fullscreen.

### Steps for Testing
Prerequisites:
- 1 Student
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Enable Splitview
4. Click on Fullscreen button and check if split view player is displayed corrently
5. Check if subtitles are displayed correctly
